robotoff/slack.py

```diff
@@ -234,26 +217,22 @@ def send_logo_notification(
             f"(<https://hunger.openfoodfacts.org/logos?logo_id={logo.id}|annotate logo>, "
             f"<{base_off_url}/product/{barcode}|product>):\n{prob_text}"
         )
-        self._post_message(text, self.ROBOTOFF_ALERT_CHANNEL)
+        self._post_message(_slack_message_block(text), self.ROBOTOFF_ALERT_CHANNEL)
 
     def _post_message(
```

Comment:
Off topic:
I think it would be better if the NoopSlackNotifier would inherit the SlackNotifier but modify _post_message to just log the payload as "info".
It would help spot eventual bugs in SlackNotifier earlier.

Reply:
Good point, I'll send a separate PR for that to keep this PR focused!